Add section 1.1.1

SuFischer opened the pull request:
final version
[main.pdf](https://github.com/jneidel/internet.org-entwicklung/files/3912895/main.pdf)

SuFischer committed:
Update bibliography.bib

ICTslowingDown habe ich den Type online gegeben, da ich nicht wusste welchen anderen Typ ich hätte nehmen können.

bei connectWorld habe ich eine Fehlermeldung

SuFischer committed:
Update main.tex

1. Ich habe das Gefühl dass ich einiges Reingepackt habe, was ich eigentlich nicht hätte reinpacken sollen.  Der erste Abschnitt hat sehr wenig mit meinem Punkt zu tuen aber ich wollte verdeutlichen, dass Facebook weiter wachsen möchte.
2. Ich bin mir bei der Bezeichnung "Entwicklungsländer" sehr unsicher, da der Dozent mal erwähnt hat dass dieser Begriff nicht mehr wirklich benutzt werden soll. Ich habe teilweise "Länder der 3. Welt" verwendet, aber eine andere Alternative fällt mir nicht ein.

jneidel committed:
Merge branch '1.1.1' of github.com:jneidel/internet.org-entwicklung into 1.1.1

jneidel commented on bibliography.bib:
```diff
@@ -69,30 +69,28 @@ @article{prasad2017
-	date            = {2016},
-	url             = {https://www.strategyand.pwc.com/gx/en/insights/2016/ctw/connecting-the-world.pdf},
+@report{connectWorld,
+  author = {Bahjat El-Darwiche, Mathias Herzog, Rawia Abdel Samad},
```
Es scheint die Menge der Autoren gewesen zu sein (weird, but ok). Nachdem ich die letzen drei rausgenommen habe erkannte er die Quelle.

jneidel committed:
Fix bibliography

jneidel commented:
See as pdf: [main.pdf](https://github.com/jneidel/internet.org-entwicklung/files/3897714/main.pdf)

jneidel commented on main.tex:
```diff
@@ -16,8 +16,36 @@
 
 \begin{document}
+\section{Facebooks Absicht/Gründe für Internet.org}
+Facebook, ein soziales Netzwerk, wurde 2004 auf den Markt gebracht und seit da an ist es, und das dahinterstehende Unternehmen, facebook inc, stetig gewachsen. 
```
Stylize fb als: `Facebook, Inc.` ([see](https://en.wikipedia.org/wiki/Facebook)

SuFischer replied:
Muss kurz nachfragen, meinst du damit, dass ich Facebook mit Facebook, ink gleichsetzen soll?

jneidel replied:
Ne nicht überall, aber `facebook inc` -> `Facebook, Inc.`

jneidel replied:
Hast du weiter unten auch schon so :)

jneidel commented on main.tex:
```diff
@@ -16,8 +16,36 @@
 \begin{document}
+\section{Facebooks Absicht/Gründe für Internet.org}
+Facebook, ein soziales Netzwerk, wurde 2004 auf den Markt gebracht und seit da an ist es, und das dahinterstehende Unternehmen, facebook inc, stetig gewachsen. 
+Doch da die jährliche Anzahl neuer Kabelanschlüsse, für das Internet, global gesehen seit 2008 immer weniger geworden ist,
```
In der westlichen Welt :)

jneidel replied:
Vlt. `...seit 2008 gesunken sind`
und `Anmeldung neuer Kabelanschlüsse`?

SuFischer replied:
Zum 1. Kommentar: schlägst du vor "global" mit "westliche Welt" auszutauschen?

jneidel replied:
> Zum 1. Kommentar: schlägst du vor "global" mit "westliche Welt" auszutauschen?

Irgendetwas in der Art, es soll halt daraus hervorgehen das in der westlichen Welt Internet schon sehr weit verbreitet ist und deshalb nicht viel Wachstumspotential besteht.

jneidel commented on main.tex:
```diff
@@ -16,8 +16,36 @@
+\section{Facebooks Absicht/Gründe für Internet.org}
+Facebook, ein soziales Netzwerk, wurde 2004 auf den Markt gebracht und seit da an ist es, und das dahinterstehende Unternehmen, facebook inc, stetig gewachsen. 
+Doch da die jährliche Anzahl neuer Kabelanschlüsse, für das Internet, global gesehen seit 2008 immer weniger geworden ist,
+(Vgl. \cite{ICTslowingDown}) hat das Wachstum von Facebook äquivalent dazu auch nachgelassen.\\
```
`äquivalent dazu` -> `dementsprechend`?

Daelerion replied:
> `äquivalent dazu` -> `dementsprechend`?

dementsprechend hört sich mM besser an.

jneidel commented on main.tex:
```diff
@@ -16,8 +16,36 @@
+Doch da die jährliche Anzahl neuer Kabelanschlüsse, für das Internet, global gesehen seit 2008 immer weniger geworden ist,
+(Vgl. \cite{ICTslowingDown}) hat das Wachstum von Facebook äquivalent dazu auch nachgelassen.\\
+Dies passiert obwohl "at the beginning of 2016, only an estimated 3.2 billion people — 44 percent of the world’s population are online and connected to the digital economy." (\cite{connectWorld})\\
+Der Grund dafür ist, dass der Großteil der restlichen 56\% aus Menschen besteht, die in den Entwicklungsländern leben. Diese haben entweder nicht die Möglichkeit Internet zu benutzen (wegen einen zu teuren/keinen Internetzugang) oder sehen nicht den Nutzen des Internets, da er ihnen nie vermittelt worden ist.
```
`die Möglichkeit Internet` -> `die Möglichkeit das Internet`

jneidel commented on main.tex:
```diff
@@ -16,8 +16,36 @@
-\maketitle
+\medskip
 
+Daraus lässt sich ableiten, dass für den weiteren Wachstum des Internets, und der Dienstleistungen auf ihm (wie zum Beispiel Facebook), es nötig ist dass das Internet in den Entwicklungsländern ausgebreitet, und die Interesse der Menschen dort dafür geweckt wird. 
```
Ich würde den Fokus eher auf das Wachstum Facebooks legen. Also: wenn Facebook wachsen möchte können sie entweder existierende Internetnutzer anwerben (nicht viele der die nicht schon auf FB sind wollen noch zu FB) oder halt neue Internetnutzer schaffen.
Es geht nicht darum das das Internet wächst, das ist nur ein Mittel zum Zweck für das Wachstum Facebooks.

jneidel replied:
Vlt. Bezug nehmen auf den Anspruch des ewigen Wirtschaftswachstums (trotz limitierter Resourcen).

jneidel commented on main.tex:
```diff
@@ -16,8 +16,36 @@
+\medskip
+
+Mark Zuckerberg, Gründer und Vorstandsvorsitzender von Facebook Inc., hat in 2013 das Dokument "Is Connectivity a Human Right?" veröffentlicht.
+In diesem erläutert er, dass es für Facebook wahrscheinlich nicht gewinnbringend sei, das Internet in den Entwicklungsländern auszubreiten. Trotzdem arbeitet Facebook daran dies zu machen, da er daran glaubt, dass jeder Mensch es verdient hat verknüpft zu sein.(Vgl. \cite{HumanRight})
```
Gibt die Quelle noch mehr zur Aussage, das es für FB nicht gewinnbringend sei, her?
Weil das ist doch das wirklich Interessante, dass eine Aktiengesellschaft hier vorgibt Philanthropie zu fahren vorgibt.

jneidel replied:
What about this?

jneidel replied:
@SuFischer und dieser Kommentar ist noch offen.

jneidel commented on main.tex:
```diff
@@ -16,8 +16,36 @@
+
+Positive Auswirkungen durch eine Ausbreitung des Internets, welches eine Bildungs- und Verbindungsmöglichkeit darstellt, in Ländern der 3. Welt, sind die Entstehung neuer Arbeitsplätze, der Rückgang der Armut und die positive Entwicklung der Wirtschaft.\\
+    
+In dem Report 'Connecting the world - Ten mechanisms for global inclusion', welcher von Facebook zur Unterstützung des Internet.org Projektes in Auftrag gegeben wurden ist, werden die globalen Auswirkungen verdeutlicht:
```
Nimm den Namen des Reports ruhig raus, der volle Titel steht ja nach dem Zitat.
I.e. mach eine generische Referenz draus.

jneidel replied:
Du führst ja den vorherigen Punkt fort (also das mehr Internet = besser für die Welt), also Binde zu ruhig den Anfang des Paragraphen an den vorherigen Inhalt an (z.B. `Facebooks Aussage würd bekräftigt durch folgenden Report...`)

jneidel commented on main.tex:
```diff
@@ -16,8 +16,36 @@
+
+\medskip
+
+Positive Auswirkungen durch eine Ausbreitung des Internets, welches eine Bildungs- und Verbindungsmöglichkeit darstellt, in Ländern der 3. Welt, sind die Entstehung neuer Arbeitsplätze, der Rückgang der Armut und die positive Entwicklung der Wirtschaft.\\
```
Quelle? Denn wenn das vom Zucc so behauptet wird, dann müsste das aus dem Text hervor gehen (z.b. Vermeintliche positive Effekte sind...)

jneidel replied:
`3. Welt` als `dritte Welt` oder wenn man den Begriff vermeiden will z.B. als `Länder ohne weitdeckende Internetverbreitung`

SuFischer replied:
Also ich war mir bei den Zitaten/Quellen die ganze Zeit sehr unsicher.
Ich wollte natürlich nicht, dass mein ganzer Text nur aus Zitaten besteht, aber ich muss mich ja eigentlich ununterbrochen auf das Beziehen was in den Dokumenten steht.
Ich dachte hier wäre es in Ordnung, das einfach im Raum stehen zu lassen, da sich 
diese Aussage ja aus dem nächsten Zitat ableiten lässt.

vielleicht wäre es eine Idee den Absatz mit den vorherigen zu verbinden.
Also `da er daran glaubt, dass jeder Mensch es verdient hat verknüpft zu sein.` 
und dann
`warum das verknüpft sein überhaupt was ist was man haben möchte => Auswirkungen`

jneidel replied:
Das verbinden der Absätze kannst du probieren!

Und es muss auch nicht an jedem Absatz eine Quelle dranstehen, aber wenn es nicht deine eigene Meinung ist, dann sollte das halt aus dem Text hervor gehen :smile:

SuFischer replied:
Ja, stimmt.
Das macht Sinn, danke.

jneidel commented on main.tex:
```diff
@@ -16,8 +16,36 @@
+    
+In dem Report 'Connecting the world - Ten mechanisms for global inclusion', welcher von Facebook zur Unterstützung des Internet.org Projektes in Auftrag gegeben wurden ist, werden die globalen Auswirkungen verdeutlicht:
+                    
+"Based on an econometric analysis of 120 countries, we believe that achieving universal Internet penetration could expand world output by \$6.7 trillion.  
```
`Based on an econometric analysis of 120 countries, ` -> `[...]` :)

jneidel replied:
Und korrekte Anführungszeichen in Latex als \`\` statt " (also nur am Anfang des Zitats).
Gilt auch für Zeile 22

jneidel commented on main.tex:
```diff
@@ -16,8 +16,36 @@
+
+\medskip
+
+Mark Zuckerbergs Meinung nach ist jedem die Möglichkeit zum verbinden zu geben die Grundlage dafür, dass eine wissensbasierte Wirtschaft erreicht wird.(\cite{HumanRight})
```
Die Formulierung hört sich sehr Englisch an :smile:

jneidel commented on main.tex:
```diff
@@ -16,8 +16,36 @@
+
+\medskip
+
+Mark Zuckerberg, Gründer und Vorstandsvorsitzender von Facebook Inc., hat in 2013 das Dokument "Is Connectivity a Human Right?" veröffentlicht.
```
`hat in 2013` -> `hat 2013` (englische Formulierung)

jneidel replied:
Erwähnung das dieses Doc die Grundlage für Internet.orgs Konzept ist

jneidel commented:
Was mir fehlt ist das hinterfragen von Facebooks Motiv, neben Philanthropie könnte noch eine alternative Erklärung beleuchtet werden.

jneidel reviewed (changes requested):
Inhaltlich gut, aber Formulierungen und ein paar andere Punkte sind noch offen :smile:

Daelerion reviewed (commented):
Jonathan hat die wichtigsten Sachen genannt, Ein paar Formulierungen sind etwas zu sehr aus dem englischem übersetzt (fühlt sich jedenfalls so an). Werde nachdem ich mein 2.1.1. geschrieben habe, mir das nocheinmal angucken, vlt kann ich mich dann etwas mehr hineinversetzen.

SuFischer committed:
Update main.tex

Version 2 (noch nicht alle comments durchgearbeitet)

Daelerion commented on main.tex:
```diff
@@ -16,8 +16,39 @@
+
+\medskip
+
+Mark Zuckerberg ist der Meinung, dass die Grundlage für eine zukünftige wissenbassierte Wirtschaft, die globale Vernetzung der Menschen ist.
```
```suggestion
Mark Zuckerberg ist der Meinung, die Grundlage für eine zukünftige wissenbassierte Wirtschaft sei die globale Vernetzung der Menschen.
```

Daelerion replied:
```suggestion
Mark Zuckerberg ist der Meinung, dass die Grundlage für eine zukünftige wissenbassierte Wirtschaft, die globale Vernetzung der Menschen sei.
```

jneidel commented on main.tex:
```diff
@@ -16,8 +16,39 @@
 }
 
 \begin{document}
+\section{Facebooks Absicht/Gründe für Internet.org}
```
```suggestion
\subsubsection{Facebooks Absicht/Gründe für Internet.org}
```

jneidel commented on main.tex:
```diff
@@ -16,8 +16,39 @@
+Doch da die Anzahl von Anmeldungen neuer Internet-Kabelanschlüsse in der westlichen Welt seit 2008 gesunken sind,
+(Vgl. \cite{ICTslowingDown}) hat das Wachstum von Facebook dementsprechent auch nachgelassen.\\
+Dies passiert obwohl ``at the beginning of 2016, only an estimated 3.2 billion people — 44 percent of the world’s population are online and connected to the digital economy." (\cite{connectWorld})\\
+Der Grund dafür ist, dass der Großteil der restlichen 56\% aus Menschen besteht, die in den Entwicklungsländern leben. Diese haben entweder nicht die Möglichkeit das Internet zu benutzen (wegen einen zu teuren/keinen Internetzugang) oder sehen nicht den Nutzen des Internets, da er ihnen nie vermittelt worden ist.
```
```suggestion
Der Grund dafür ist, dass der Großteil der restlichen 56\% aus Menschen besteht, die in Entwicklungsländern leben. Diese haben entweder nicht die Möglichkeit das Internet zu benutzen (wegen einen zu teuren/keinen Internetzugang) oder sehen nicht den Nutzen des Internets, da er ihnen nie vermittelt worden ist.
```

jneidel commented on main.tex:
```diff
@@ -16,8 +16,39 @@
+Daraus lässt sich ableiten, dass für den weiteren Wachstum Facebooks es nötig ist, dass das Internet in den Entwicklungsländern ausgebreitet, und die Interesse der Menschen dort für die Anwendung geweckt wird
+
+ @jneidel
+jneidel 3 hours ago Owner
```
```suggestion
```

jneidel commented on main.tex:
```diff
@@ -16,8 +16,39 @@
 
+Daraus lässt sich ableiten, dass für den weiteren Wachstum Facebooks es nötig ist, dass das Internet in den Entwicklungsländern ausgebreitet, und die Interesse der Menschen dort für die Anwendung geweckt wird
+
+ @jneidel
```
```suggestion
```

SuFischer committed:
Update main.tex

Version 3

jneidel commented on main.tex:
```diff
@@ -16,8 +16,37 @@
 }
 
 \begin{document}
+\subsection{Facebooks Absicht/Gründe für Internet.org}
```
```suggestion
\subsection{Beweggründe für Internet.org}
```

Gefällt mir besser als Absichten

jneidel commented on main.tex:
```diff
@@ -16,8 +16,37 @@
 
 \begin{document}
+\subsection{Facebooks Absicht/Gründe für Internet.org}
+Facebook, ein soziales Netzwerk, wurde 2004 auf den Markt gebracht und seit da an ist es, und das dahinterstehende Unternehmen Facebook, Inc., stetig gewachsen. 
```
```suggestion
Facebook, ein soziales Netzwerk, wurde 2004 gegründet und ist seit dem, genauso wie das dahinterstehende Unternehmen Facebook, Inc., stetig gewachsen. 
```

jneidel commented on main.tex:
```diff
@@ -16,8 +16,37 @@
+Doch da die Anzahl von Anmeldungen neuer Internet-Kabelanschlüsse in der westlichen Welt seit 2008 gesunken sind,
+(Vgl. \cite{ICTslowingDown}) hat das Wachstum von Facebook dementsprechent auch nachgelassen.\\
+Dies passiert obwohl ``at the beginning of 2016, only an estimated 3.2 billion people — 44 percent of the world’s population are online and connected to the digital economy." (\cite{connectWorld})\\
+Der Grund dafür ist, dass der Großteil der restlichen 56\% aus Menschen besteht, die in Entwicklungsländern leben. Diese haben entweder nicht die Möglichkeit das Internet zu benutzen (wegen einen zu teuren/keinen Internetzugang) oder sehen nicht den Nutzen des Internets, da er ihnen nie vermittelt worden ist.
```
```suggestion
Der Grund dafür: ein der Großteil der restlichen 56\% von Menschen lebt in Entwicklungsländern. Diese haben entweder nicht die Möglichkeit das Internet zu benutzen (wegen einen zu teuren/keinen Internetzugang) oder sehen nicht den Nutzen des Internets, da er ihnen nie vermittelt worden ist.
```
Hoffe das klingt nicht zu lapidar :smile:

jneidel replied:
```suggestion
Der Grund dafür ist, dass der Großteil der restlichen 56\% aus Menschen besteht, die in Entwicklungsländern leben. Diese haben entweder nicht die Möglichkeit das Internet zu nutzen (wegen zu hoher Kosten/keinen Internetzugang) oder sehen nicht den Nutzen des Internets, da er ihnen nie vermittelt worden ist.
```

jneidel replied:
```suggestion
Der Grund dafür ist, dass der Großteil der restlichen 56\% aus Menschen besteht, die in Entwicklungsländern leben. Diese haben entweder nicht die Möglichkeit das Internet zu benutzen (wegen einen zu teuren/keinen Internetzugang) oder sehen nicht den Nutzen des Internets, da ihnen dieser nie vermittelt worden ist.
```

jneidel reviewed (changes requested):
Im hinteren Teil bleiben noch die ein, zwei inhaltlichen Sachen aus dem letzen Review bestehen.

Formulierungen schaue ich mir für den Teil an sobald der Inhalt fix ist :)

SuFischer committed:
Update main.tex

Co-Authored-By: Jonathan Neidel <github@jneidel.com>

SuFischer committed:
Update main.tex

Co-Authored-By: Jonathan Neidel <github@jneidel.com>

SuFischer committed:
Update main.tex

Co-Authored-By: Jonathan Neidel <github@jneidel.com>

SuFischer committed:
Update main.tex

Korrekturen + kritischerer Blick
-Bei den kritischen Blick muss ich wahrscheinlich noch ne Quelle hinzufügen
-Mit Zeile 26 bin ich immer noch unzufrieden 😅

jneidel commented:
Kannst du noch Facebook Zero als Basis für Internet.org integrieren (gleiche Idee, anderer Scope).

SuFischer commented:
Facebook Zero habe ich am Ende der Umsetzung kurz erwähnt.

Ich fand es dort passender, da sie ja Internet.org umsetzen, indem sie die Technologie benutzen die sie bereits bei Facebook Zero im kleineren Umfang  angewandt haben.

SuFischer committed:
Update main.tex

kleine Veränderungen.
Frage: seid ihr der Meinung dass nach <...., sehen einige Journalsiten hierbei ein Rennen für die Internetverbreitung.> 
eine Quelle von einem Beispiel notwendig ist?

SuFischer committed:
Update bibliography.bib

Hinzufügen einer Quelle

jneidel commented on main.tex:
```diff
@@ -16,8 +16,43 @@
 \begin{document}
+\subsection{Beweggründe für Internet.org}
+Facebook, ein soziales Netzwerk, wurde 2004 gegründet und ist seit dem, genauso wie das dahinterstehende Unternehmen Facebook, Inc., stetig gewachsen. 
+Doch da die Anzahl von Anmeldungen neuer Internet-Kabelanschlüsse in der westlichen Welt seit 2008 gesunken sind,
```
```suggestion
Doch da die Anzahl von Anmeldungen neuer Internet-Kabelanschlüsse in der westlichen Welt seit 2008 gesunken sind
```

jneidel commented on main.tex:
```diff
@@ -16,8 +16,43 @@
+\subsection{Beweggründe für Internet.org}
+Facebook, ein soziales Netzwerk, wurde 2004 gegründet und ist seit dem, genauso wie das dahinterstehende Unternehmen Facebook, Inc., stetig gewachsen. 
+Doch da die Anzahl von Anmeldungen neuer Internet-Kabelanschlüsse in der westlichen Welt seit 2008 gesunken sind,
+(Vgl. \cite{ICTslowingDown}) hat das Wachstum von Facebook dementsprechent auch nachgelassen.\\
```
```suggestion
(Vgl. \cite{ICTslowingDown}), hat das Wachstum von Facebook dementsprechent auch nachgelassen.\\
```

jneidel commented on main.tex:
```diff
@@ -16,8 +16,43 @@
+Doch da die Anzahl von Anmeldungen neuer Internet-Kabelanschlüsse in der westlichen Welt seit 2008 gesunken sind,
+(Vgl. \cite{ICTslowingDown}) hat das Wachstum von Facebook dementsprechent auch nachgelassen.\\
+Dies passiert obwohl ``at the beginning of 2016, only an estimated 3.2 billion people — 44 percent of the world’s population are online and connected to the digital economy." (\cite{connectWorld})\\
+Der Grund dafür: der Großteil der restlichen 56\% der Menschen lebt in Entwicklungsländern. Dort hat man meist entweder nicht die Möglichkeit das Internet zu nutzen (wegen zu hoher Kosten/keinen Internetzugang) oder man sieht den Nutzen des Internets nicht, da dieser nie vermittelt worden ist.
```
```suggestion
Der Grund dafür: der Großteil der restlichen 56\% der Menschen lebt in Entwicklungsländern. Dort besteht meist entweder nicht die Möglichkeit das Internet zu nutzen (wegen zu hoher Kosten/keinem Internetzugang) oder man sieht den Nutzen des Internets nicht, da einem dieser nie vermittelt worden ist.
```

jneidel commented on main.tex:
```diff
@@ -16,8 +16,43 @@
-\maketitle
+\medskip
 
+Für Facebook, eine Firma welche durch Werbung und Personenbezogenen Daten Geld verdient, ist es von langfristiger Interesse, dass sich die Anzahl von Menschen, welche das Internet benutzen, erhöht.
```
```suggestion
Für Facebook, eine Firma welche durch Werbung und personenbezogene Daten Geld verdient, ist es von langfristigem Interesse, dass sich die Anzahl der Internetnutzer erhöht.
```

jneidel commented on main.tex:
```diff
@@ -16,8 +16,43 @@
+\medskip
 
+Für Facebook, eine Firma welche durch Werbung und Personenbezogenen Daten Geld verdient, ist es von langfristiger Interesse, dass sich die Anzahl von Menschen, welche das Internet benutzen, erhöht.
+Aus den vorherigen Pragraph lässt sich ableiten, dass sich dafür das Internet in den Entwicklungsländern ausbreiten, und die Interesse der Menschen für die Anwendung geweckt werden muss.
```
```suggestion
Es lässt sich ableiten, dass dafür das Internet in den Entwicklungsländern ausbreitet und das Interesse der Menschen für die Anwendung geweckt werden muss.
```

jneidel commented on main.tex:
```diff
@@ -16,8 +16,43 @@
 
+Für Facebook, eine Firma welche durch Werbung und Personenbezogenen Daten Geld verdient, ist es von langfristiger Interesse, dass sich die Anzahl von Menschen, welche das Internet benutzen, erhöht.
+Aus den vorherigen Pragraph lässt sich ableiten, dass sich dafür das Internet in den Entwicklungsländern ausbreiten, und die Interesse der Menschen für die Anwendung geweckt werden muss.
+Facebook ist nicht die einzige IT-Firma welche Interesse daran gezeigt hat.
```
```suggestion
Facebook ist nicht die einzige IT-Firma welche daran Interesse  zeigt.
```

jneidel commented on main.tex:
```diff
@@ -16,8 +16,43 @@
+Für Facebook, eine Firma welche durch Werbung und Personenbezogenen Daten Geld verdient, ist es von langfristiger Interesse, dass sich die Anzahl von Menschen, welche das Internet benutzen, erhöht.
+Aus den vorherigen Pragraph lässt sich ableiten, dass sich dafür das Internet in den Entwicklungsländern ausbreiten, und die Interesse der Menschen für die Anwendung geweckt werden muss.
+Facebook ist nicht die einzige IT-Firma welche Interesse daran gezeigt hat.
+Eine weitere ist die Dachgesellschaft von Google, sie hat 2011 mit ihren Tests für das Projekt Loon, welches das Ziel hat das Internet Netzwerk auf abgelegende Gegenden auszubreiten, angefangen.(Vgl. \cite{projectLoon})
```
```suggestion
Eine weitere ist die Dachgesellschaft von Google, Alphabet Inc., welche mit dem Projekt Loon das Ziel verfolgt Internet in abgelegenden Gegenden auszubreiten (Vgl. \cite{projectLoon}).
```

jneidel replied:
Anstatt Projekt Loon, kann vlt. lieber Bezug auf [googles RailWire](https://gadgets.ndtv.com/internet/features/google-railwire-railtel-india-free-internet-train-stations-1731879) (auch in Indien) genommen werden.

jneidel replied:
Das wird nämlich heute auch benutzt

jneidel commented on main.tex:
```diff
@@ -16,8 +16,43 @@
+Aus den vorherigen Pragraph lässt sich ableiten, dass sich dafür das Internet in den Entwicklungsländern ausbreiten, und die Interesse der Menschen für die Anwendung geweckt werden muss.
+Facebook ist nicht die einzige IT-Firma welche Interesse daran gezeigt hat.
+Eine weitere ist die Dachgesellschaft von Google, sie hat 2011 mit ihren Tests für das Projekt Loon, welches das Ziel hat das Internet Netzwerk auf abgelegende Gegenden auszubreiten, angefangen.(Vgl. \cite{projectLoon})
+Obwohl sie unterschiedliche Herngehensweisen und Schwerpunkte haben, sehen einige Journalsiten hierbei ein Rennen für die Internetverbreitung. \\
```
```suggestion
Obwohl sie unterschiedliche Herngehensweisen und Schwerpunkte haben, sehen einige Journalsiten hierbei ein Rennen der Internetverbreitung. \\
```

jneidel commented on main.tex:
```diff
@@ -16,8 +16,43 @@
+Eine weitere ist die Dachgesellschaft von Google, sie hat 2011 mit ihren Tests für das Projekt Loon, welches das Ziel hat das Internet Netzwerk auf abgelegende Gegenden auszubreiten, angefangen.(Vgl. \cite{projectLoon})
+Obwohl sie unterschiedliche Herngehensweisen und Schwerpunkte haben, sehen einige Journalsiten hierbei ein Rennen für die Internetverbreitung. \\
+    
+Im folgenden Text wird auf die Gründe die Facebook angegeben hat eingegangen, doch diese Gründe werden kritisch gesehen.   
```
```suggestion
In den folgenden Paragraphen für auf Facebooks Beweggründe eingegangen, doch diese Gründe werden kritisch gesehen.
```

jneidel commented on main.tex:
```diff
@@ -16,8 +16,43 @@
+Im folgenden Text wird auf die Gründe die Facebook angegeben hat eingegangen, doch diese Gründe werden kritisch gesehen.   
+
+\medskip
+Mark Zuckerberg, Gründer und Vorstandsvorsitzender von Facebook Inc., hat 2013 das Dokument "Is Connectivity a Human Right?" veröffentlicht.
```
```suggestion
Mark Zuckerberg, Gründer und Vorstandsvorsitzender von Facebook Inc., hat 2013 das Dokument ``Is Connectivity a Human Right?" veröffentlicht.
```

jneidel commented on main.tex:
```diff
@@ -16,8 +16,43 @@
+
+\medskip
+Mark Zuckerberg, Gründer und Vorstandsvorsitzender von Facebook Inc., hat 2013 das Dokument "Is Connectivity a Human Right?" veröffentlicht.
+In diesem, welches die Grundlage für Internet.orgs Konzept ist, erläutert er, dass es für Facebook wahrscheinlich nicht gewinnbringend sei, das Internet in den Entwicklungsländern auszubreiten. Trotzdem arbeitet Facebook daran dies zu machen, da er daran glaubt, dass jeder Mensch es verdient hat verknüpft zu sein.(Vgl. \cite{HumanRight})
```
```suggestion
Das Dokument stellt die Grundlage für Internet.orgs Konzept dar. Es erläutert das es für Facebook wahrscheinlich nicht gewinnbringend sei, das Internet in Entwicklungsländern auszubreiten. Trotzdem möchte  Facebook daran arbeitet, da er glaubt, dass jeder Mensch es verdient hat verknüpft zu sein.(Vgl. \cite{HumanRight})
```

jneidel replied:
`verdient hat verknüpft zu sein` klingt sehr englisch

jneidel commented on main.tex:
```diff
@@ -16,8 +16,43 @@
+\medskip
+
+Angebliche positive Auswirkungen durch eine Ausbreitung des Internets, welches eine Bildungs- und Verbindungsmöglichkeit darstellt, in Ländern der dritten Welt, seien die Entstehung neuer Arbeitsplätze, der Rückgang der Armut und die positive Entwicklung der Wirtschaft.\\
+    
```
```suggestion
```

jneidel commented on main.tex:
```diff
@@ -16,8 +16,43 @@
+
+\medskip
+
+Angebliche positive Auswirkungen durch eine Ausbreitung des Internets, welches eine Bildungs- und Verbindungsmöglichkeit darstellt, in Ländern der dritten Welt, seien die Entstehung neuer Arbeitsplätze, der Rückgang der Armut und die positive Entwicklung der Wirtschaft.\\
```
```suggestion
Angebliche positive Auswirkungen durch eine Ausbreitung des Internets, welches eine Bildungs- und Verbindungsmöglichkeit darstellt, in Ländern der dritten Welt, seien die Entstehung neuer Arbeitsplätze, der Rückgang der Armut und die positive Entwicklung der Wirtschaft.
```

jneidel commented on main.tex:
```diff
@@ -16,8 +16,43 @@
+
+Angebliche positive Auswirkungen durch eine Ausbreitung des Internets, welches eine Bildungs- und Verbindungsmöglichkeit darstellt, in Ländern der dritten Welt, seien die Entstehung neuer Arbeitsplätze, der Rückgang der Armut und die positive Entwicklung der Wirtschaft.\\
+    
+Diese Auswirkungen werden in einem Report, welcher von Facebook zur Unterstützung des Internet.org Projektes in Auftrag gegeben wurden ist, global verdeutlicht:
```
```suggestion
Diese Auswirkungen werden in einem von Facebook in Auftrag gegebenen Report verdeutlicht:
```

jneidel commented on main.tex:
```diff
@@ -16,8 +16,43 @@
+``[...] we believe that achieving universal Internet penetration could expand world output by \$6.7 trillion.  
+In addition, global inclusion could bring 7 percent of the world’s population above absolute poverty levels — in effect providing 500 
+million of the world’s poorest people new opportunities to grow and engage with the world"
+\parencite{connectWorld}
```
```suggestion
- \cite{connectWorld}
```

jneidel commented on main.tex:
```diff
@@ -16,8 +16,43 @@
+
+\medskip
+
+Mark Zuckerberg ist der Meinung, dass die Grundlage für eine zukünftige wissenbassierte Wirtschaft, die globale Vernetzung der Menschen sei.
```
```suggestion
Mark Zuckerberg ist weiterhin der Meinung, dass die Grundlage für eine zukünftige wissenbassierte Wirtschaft, die globale Vernetzung der Menschheit sei \parencite{HumanRight}.
```

jneidel commented on main.tex:
```diff
@@ -16,8 +16,43 @@
+\medskip
+
+Mark Zuckerberg ist der Meinung, dass die Grundlage für eine zukünftige wissenbassierte Wirtschaft, die globale Vernetzung der Menschen sei.
+(\cite{HumanRight})
```
```suggestion
```

jneidel reviewed (changes requested):
Besser!

Sind nur noch ein paar Formulierungen offen.

SuFischer committed:
Update main.tex

Co-Authored-By: Jonathan Neidel <github@jneidel.com>

SuFischer committed:
Update main.tex

Co-Authored-By: Jonathan Neidel <github@jneidel.com>

SuFischer committed:
Update main.tex

Co-Authored-By: Jonathan Neidel <github@jneidel.com>

SuFischer committed:
Update main.tex

Co-Authored-By: Jonathan Neidel <github@jneidel.com>

SuFischer committed:
Update main.tex

Co-Authored-By: Jonathan Neidel <github@jneidel.com>

SuFischer committed:
Update main.tex

Co-Authored-By: Jonathan Neidel <github@jneidel.com>

SuFischer committed:
Update main.tex

Co-Authored-By: Jonathan Neidel <github@jneidel.com>

SuFischer committed:
Update main.tex

Co-Authored-By: Jonathan Neidel <github@jneidel.com>

SuFischer committed:
Update main.tex

Co-Authored-By: Jonathan Neidel <github@jneidel.com>

SuFischer committed:
Update main.tex

Co-Authored-By: Jonathan Neidel <github@jneidel.com>

SuFischer committed:
Update main.tex

Formatierung und minimale Veränderung

SuFischer committed:
Update main.tex

SuFischer committed:
Update bibliography.bib

SuFischer committed:
Update main.tex

spezifische Seiten bei den Quellen angegeben

jneidel committed:
Update main.tex

jneidel committed:
Update main.tex

jneidel committed:
Proofread

jneidel committed:
Merge branch 'master' into 1.1.1

jneidel committed:
Proofread
